.idea/jarRepositories.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Untrack this file in your git, it's a generated file and gets regenerated all the time.

Reply:
will do